### PR TITLE
TELCODOCS-1707: Add xrefs

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.16+

Issue: https://issues.redhat.com/browse/TELCODOCS-1707

Link to docs preview: 
https://77947--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.html
https://77947--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.html
https://77947--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-generate-seed.html
https://77947--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-install-operators.html
https://77947--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-prep-resources.html
https://77947--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-image.html
https://77947--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.html
https://77947--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/ztp-image-based-upgrade.html

QE review:
- [ ] QE has approved this change.

Additional information: Last commit from the original [PR](https://github.com/openshift/openshift-docs/pull/74025)

### CODE REVIEW

File: edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc [hunk 1]
```diff
@@ -13,15 +13,18 @@ You update this CR to specify the image repository of the seed image and to move
...
-* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc#ztp-image-based-upgrade-creating-backup-resources-with-ztp_ztp-gitops[Creating ConfigMap objects for the image-based upgrade with Lifecycle Agent]
-////
+* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-prep-resources.adoc#cnf-image-based-upgrade-prep-resources[Creating ConfigMap objects for the image-based upgrade with Lifecycle Agent]
 
 include::modules/cnf-image-based-upgrade-with-backup.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#ztp-image-based-upgrade-rollback_cnf-image-based-upgrade[(Optional) Moving to the Rollback stage of the image-based upgrade with Lifecycle Agent]
```
Comment (maxwelldb) [inline edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc:26]: No error here, but I am surprised to see modules with `(Optional)` prepended to their titles in the docs. That shouldn't be making it through reviews.

File: edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc [hunk 2]
```diff
@@ -10,16 +10,18 @@ Create your OADP resources, extra manifests, and custom catalog sources wrapped
...
 .Additional resources
 
 * xref:../../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-image.adoc#ztp-image-based-upgrade-shared-container-directory_shared-container-directory[Configuring a shared container directory between ostree stateroots when using GitOps ZTP]
 
-* xref:../../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-install-operators#zp-image-based-upgrade-installing-oadp-with-gitops_install-operators[Installing and configuring the OADP Operator with GitOps ZTP]
-
-* xref:../../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent and GitOps ZTP]
-////
\ No newline at end of file
+* xref:../../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent and GitOps ZTP]
```
Comment (maxwelldb) [inline edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc:27]: Note: The page title differs from the link title.

**Review by maxwelldb — COMMENTED** [review]
I think this is largely fine. Left one note to verify about link text + page title agreement.

File: edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc (hunk 1 above)
Comment (amolnar-rh) [reply edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc:26]: I will need to open a post-GA PR to add a few comments, I'll make sure to update these.

**Comment by stevsmit** [comment]
In this assembly https://77947--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/ztp-image-based-upgrade#ztp-image-based-upgrade-prep-gitops_ztp-image-based-upgrade, you mention "Create policies and ConfigMap objects for resources used in the image-based upgrade. For more information, see "Creating ConfigMap objects for the image-based upgrade with GitOps ZTP", but I don't see that link anywhere on the page. It's also missing an end quotation. See preview for what I'm talking about.

I am assuming that it's one of these, just slightly mis (or re-) named (or maybe it's something else entirely)

[Creating ConfigMap objects for the image-based upgrade with Lifecycle Agent using GitOps ZTP]

[Configuring a shared container directory between ostree stateroots when using GitOps ZTP]

File: edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-prep-resources.adoc [hunk 3]
```diff
@@ -10,19 +10,20 @@ The {lcao} needs all your OADP resources, extra manifests, and custom catalog so
...
+include::modules/cnf-image-based-upgrade-prep-catalogsource.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
 
 * xref:../../../operators/understanding/olm/olm-understanding-olm.adoc#olm-catalogsource_olm-understanding-olm[Catalog source]
-////
\ No newline at end of file
+
+* xref:../../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#cnf-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent]
```
Comment (stevsmit) [inline edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-prep-resources.adoc:29]: Links to "Performing an image-based upgrade for single-node OpenShift clusters" if you want to fix the friend reference text post GA

**Comment by stevsmit** [comment]
@amolnar-rh A few nits to consider post-GA or in your next PR.

**Pull request merged**